bionic/dagviz.py

```diff
@@ -53,7 +54,7 @@ def show(self):
 
     def _repr_svg_(self):
         """Rich display image as SVG in IPython notebook or Qt console."""
-        return str(self._xml_text)
+        return self._xml_text.decode("utf8")
```

Comment:
Nice find, this fix was easy enough! Q: why don't we pass encoding to `pydot_graph.create_svg()` instead of decoding it here?

Reply:
Thanks, this is a good point. I don't think there's any argument to `create_svg()` that will make it return a string, but I do think it's weird to have an `_xml_text` field that's actually bytes. I decided to keep it as bytes (since we also send it to file objects) but rename it.